src/Resources/config/services.yml

```diff
@@ -430,6 +430,26 @@ services:
             - "@contao.framework"
             - "@logger"
 
+    contao.security.two_factor.authenticator:
+        class: Contao\CoreBundle\Security\TwoFactor\ContaoTwoFactorAuthenticator
+        public: true
+
+    contao.security.two_factor.backend.form_renderer:
+        class: Contao\CoreBundle\Security\TwoFactor\ContaoBackendTwoFactorFormRenderer
+        arguments:
+            - "@router"
+        public: false
```

Comment:
This is not needed, as services are private by default.

Reply:
👍